stl/inc/system_error

```diff
@@ -483,18 +497,13 @@ public:
     }
 
     _NODISCARD virtual string message(int _Errcode) const override {
-        const unsigned long _Size = 32767;
-        string _Narrow(_Size, '\0');
-
-        const unsigned long _Val = _Winerror_message(static_cast<unsigned long>(_Errcode), &_Narrow[0], _Size);
-        if (_Val == 0) {
-            _Narrow = "unknown error";
+        _System_error_message _Msg(static_cast<unsigned long>(_Errcode));
+        if (_Msg._Length == 0) {
+            static constexpr char _Unknown_error[] = "unknown error";
+            return string(_Unknown_error, _STD size(_Unknown_error) - 1);

```

Comment:
I observe that you could store `_STD size(_Unknown_error) - 1` as a `constexpr` variable, instead of calculating it at runtime, which would be an extremely minor improvement for debug codegen size.